### ISSUE TITLE
Deployed site missing UserContext

### ISSUE THREAD

**Issue by nnelgxorz** [BuilderIO/qwik#838]
### Qwik Version

0.0.38

### Operating System (or Browser)

Debian Linux

### Which component is affected?

Qwik City

### Expected Behaviour

A deployed site should work like any other site.

### Actual Behaviour

The screen is blank and there are errors in the Netlify edge function logs. This seems to be the issue Ryan and Manu hit on the Qwik City stream.

### Additional Information

The followng errors are logged in the Netlify edge function logs:
10:41:16 AM: QWIK ERROR Error: Missing Qwik City User Context
10:41:16 AM: QWIK ERROR Error: Code(13): not found state for useContext

Deploying to Netlify was not an issue before the Vite 3 updates, so I'm wondering is something changed with builds.

I'd also be curious if the same issue is happening with the new Docs site, before digging into any Netlify specific code.

**Comment by manucorporat** [comment]
i am looking into this today! hopefully an easy fix! thanks for opening an issue

**Comment by nnelgxorz** [comment]
Thanks @manucorporat!

**Comment by michaelbromley** [comment]
I think I'm running into the same when trying to deploy to CF Pages. I get a blank page at https://fcade2b3.storefront-qwik-starter.pages.dev/ and when I try to debug locally using wrangler, I get the following stack trace:

```
Error: Missing Qwik City User Context
    at C:\Users\Development\vendure\storefront-qwik-starter\server\entry.cloudflare.js:94:13
    at useStore (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\use\use-store.public.js:74:46)
    at C:\Users\Development\vendure\storefront-qwik-starter\server\entry.cloudflare.js:91:22
    at useInvoke (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\use\use-core.js:51:26)
    at C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\import\qrl-class.js:58:28
    at then (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\util\promises.js:19:56)
    at C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\import\qrl-class.js:48:20
    at C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\render\render-component.js:31:27
    at safeCall (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\util\promises.js:6:25)
    at renderComponent (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\render\render-component.js:31:12)
    at patchVnode (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\render\cursor.js:238:33)
    at updateChildren (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\render\cursor.js:82:26)
    at smartUpdateChildren (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\render\cursor.js:46:16)
    at visitJsxNode (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\render\render.js:17:16)
    at renderRoot (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\render\render.public.js:65:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at render (C:\Users\Development\vendure\storefront-qwik-starter\dist-dev\tsc-out\packages\qwik\src\core\render\render.public.js:44:23)
    at renderToStream (C:\Users\Development\vendure\storefront-qwik-starter\node_modules\@builder.io\qwik\server.mjs:9447:3)
    at renderToString (C:\Users\Development\vendure\storefront-qwik-starter\node_modules\@builder.io\qwik\server.mjs:9510:18)
    at onRequest (C:\Users\Development\vendure\storefront-qwik-starter\server\entry.cloudflare.js:248:20)
    at next (C:\Users\Development\vendure\storefront-qwik-starter\node_modules\wrangler\templates\pages-template-worker.ts:129:22)
    at ServiceWorkerGlobalScope.[kDispatchFetch] (C:\Development\vendure\storefront-qwik-starter\node_modules\@miniflare\core\src\standards\event.ts:359:13)
    at Server.<anonymous> (C:\Development\vendure\storefront-qwik-starter\node_modules\@miniflare\http-server\src\index.ts:290:20)
```

**Comment by nnelgxorz** [comment]
@michaelbromley Yep. That's the one.

**Comment by gioboa** [comment]
Btw now we updated the config files same as QwikCity cloudflare starter. Unfortunately now we have a different error inside renderToStream function 😅
<img width="510" alt="image" src="https://user-images.githubusercontent.com/35845425/181621785-f3be2e35-0a6e-4561-9043-d0b7587e5d00.png">

**Comment by nnelgxorz** [comment]
Fixed in Qwik City 0.0.29
